addons/website/static/src/snippets/s_image_gallery/gallery_slider.js

```diff
@@ -71,6 +95,17 @@ export class GallerySlider extends Interaction {
         this.onSlidCarousel();
     }
 
+    start() {
+        if (this.carouselItemEls) {
+            const firstImageEl = this.carouselItemEls.item(0).firstElementChild;
```

Comment:
We should maybe looking for an image element here, in case a custo adds another element before the image (for example a caption)